ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1149,6 +1150,40 @@ interface_groups:
       - "< profile_name >"
 ```
 
+#### Profiles and units
+```yaml
+maintenance:
+  default_interface_profile: < interface_profile_1 >
+  default_bgp_profile: < bgp_profile_1 >
+  default_unit_profile: < unit_profile_1 >
+  interface_profiles:
+    < interface_profile_1 >:
+      rate_monitoring:
+        load_interval: < seconds >
+        threshold: < kbps >
+      shutdown:
+        max_delay: < seconds >
+  bgp_profiles:
+    < bgp_profile_1 >:
+      initiator:
+        route_map_inout: < route_map >
+  unit_profiles:
+    < unit_profile_1 >:
+      on_boot:
+        duration: < 300-3600 >
+  units:
+    < unit_name_1 >:
+      quiesce: < true | false >
+      profile: < unit_profile_1 >
+      groups:
```

Comment:
This level is no longer needed after you convinced us of using `bgp_groups` and `interface_groups`. So I suggest that we flatten it a bit by removing this level.